packages/backend/src/tests/tigerbeetle.ts

```diff
@@ -31,11 +31,12 @@ export async function startTigerbeetleContainer(clusterId?: number): Promise<{
       'format',
       '--cluster=' + tigerbeetleClusterId,
       '--replica=0',
+      '--replica-count=1',
       `${TIGERBEETLE_DIR}/${tigerbeetleFile}`
     ])
     .withWaitStrategy(
       Wait.forLogMessage(
-        `info(main): 0: formatted: cluster=${tigerbeetleClusterId}`
+        `info(main): 0: formatted: cluster=${tigerbeetleClusterId} replica_count=1`
```

Comment:
is `replica_count` new?

Reply:
Yes. Took me forever to figure that out. The docs on in the TB Readme also weren't updated properly. I pinged the team about that.